src/App.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Search from './components/Search';
+import Footer from './components/Footer';
+import './App.css';
+
+const App = () => {
+  return (
+    <>
+      <div className="app flex flex-col items-center p-8">
```

Comment:
[minor] You could use `main` instead of `div` here, which is better semantically.